src/AgGrid.elm

```diff
@@ -286,6 +307,7 @@ type alias GridConfig =
             }
     , disableResizeOnScroll : Bool
     , filterStates : Dict.Dict String FilterState
+    , groupDefaultExpanded : Int
```

Comment:
I don't like this name but I don't know if I like it more if we deviate from the ag-grid docs.

Reply:
I try to stick to the ag-grid configuration names for the attributes as much as possible to make it easier for users to look up the details in the ag-grid docs. This is very helpful IMO, especially since we do not provide any documentation for the configuration ourselves.